llama_index/vector_stores/faiss.py

```diff
@@ -114,6 +123,10 @@ def persist(
             persist_path (str): The save_path of the file.
 
         """
+        # I don't think FAISS supports fsspec, it requires a path in the SWIG interface
+        # TODO: write to a temporary file and then copy to the final destination
```

Comment:
would be great to have the support here as well! But no worries if you can't get to it, we can add it later as well.